cosmovisor/cmd/cosmovisor/init.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/cosmovisor"
+	cverrors "github.com/cosmos/cosmos-sdk/cosmovisor/errors"
+)
+
+func init() {
+	rootCmd.AddCommand(initCmd)
+}
+
+var initCmd = &cobra.Command{
+	Use:          "init <path to executable>",
+	Short:        "Initializes a cosmovisor daemon home directory.",
+	Args:         cobra.ExactArgs(1),
+	SilenceUsage: true,
```

Comment:
```suggestion
	SilenceUsage: false,
```

I wouldn't silence the usage here. Otherwise, you don't know which argument you must provide which makes the error message unclear.